src/Countries/US/Indiana/IndianaLocalIncome.php

```diff
@@ -242,7 +241,11 @@ public function doesApply(Collection $tax_areas): bool
 
     public function compute(Collection $tax_areas)
     {
-        $this->tax_total = $this->payroll->withholdTax(($this->getAdjustedEarnings() * $this->getTaxRate() / $this->payroll->pay_periods) + $this->tax_information->additional_county_withholding);
+        if ($this->getAdjustedEarnings() > 0) {
+            $this->tax_total = $this->payroll->withholdTax(($this->getAdjustedEarnings() / $this->payroll->pay_periods) + $this->tax_information->additional_county_withholding);
+        } else {
+            $this->tax_total = $this->payroll->withholdTax($this->tax_information->additional_county_withholding);
```

Comment:
what if they done have enough to cover the additional county withholding?

Reply:
The `withholdTax` function takes care of that. I am 98% sure this is how we always do additional withholding. In the `withholdTax` it gets the lower of `getNetEarnings` and the `amount` passed in.